### DISCUSSION TITLE
Inquiry Regarding Flexibility in Customizing Hero Banners

### DISCUSSION THREAD

**Discussion by user12813** [TDRA-ae/aegov-dls#55] [Q&A]
I have a question about the flexibility of using UAE Design System. I want to add a hero Banner in a Federal Website, am I fixed to use only the option mentioned in https://designsystem.gov.ae/docs/blocks/hero or I can have more flexibility to change. for example I liked the "Simple centered" in tailwind https://tailwindui.com/components/marketing/sections/heroes. the closest one to this from UAE Design System is the "Hero static" banner

**Comment by moiz-conceptlz** [comment]
@user12813 

The "simple centered" is indeed very close to the "Hero Static". The blocks used in the UAE Design System are designed based on feedback from govt entities.

However, we are very open to suggestions and improvements. 

Share your design, an if it is indeed useful to have this it can be considered as a permanent addition and created as a block.
